pdns/dnsdistdist/dnsdist-lua-hooks.cc

```diff
@@ -7,7 +7,7 @@
 namespace dnsdist::lua::hooks
 {
 using MaintenanceCallback = std::function<void()>;
-using TicketsKeyAddedHook = std::function<void(const char*, size_t)>;
+using TicketsKeyAddedHook = std::function<void(const std::string&, size_t)>;
```

Comment:
Does it still make sense to pass the size separately then?

Reply:
It's not necessary, but the API has been like that in two released versions already so let's keep it.